config/initializers/postgres_required_versions.rb

```diff
@@ -5,14 +5,13 @@ def initialize(*args)
   end
 
   def check_version
-    msg = "The version of PostgreSQL being connected to is incompatible with #{Vmdb::Appliance.PRODUCT_NAME} (10 required)"
+    msg = "The version of PostgreSQL being connected to (#{postgresql_version}) is incompatible with #{Vmdb::Appliance.PRODUCT_NAME} (130000 / 13 required)"
```

Comment:
I always thought it was weird it didn't report what version you're on.  There isn't a convenient method that returns just the 13.0 string, so I'm returning the value straight from this method:  13000, 12000, etc.